src/components/DynamicCalendar/index.js

```diff
@@ -292,9 +292,14 @@ class DynamicCalendar extends Component {
 
             markedDatesArray.push(formattedStartDate)
           }
-          const dates = this.getDates(new Date(startTime), new Date(endTime))
+          // new Date(endTime) is returning undefined here \/
+          const dates = this.getDates(
+            moment(startTime, 'YYYY-MM-DD').toDate(),
+            moment(endTime, 'YYYY-MM-DD').toDate()
+          )
+
           // Adds other marked dates (starts with 2 to avoid first day)
```

Comment:
@TDH55 could you just fix/remove the comment?

```suggestion
          // Adds other marked dates (starts with 1 to avoid first day)
```